api/lib/utils.js

```diff
@@ -115,6 +117,34 @@ function isValidLedgerWord(ledger) {
   return (/^current$|^closed$|^validated$/.test(ledger));
 }
 
+function attachDate(api, baseTransactions, callback) {
+  var groupedTx = _.groupBy(baseTransactions, function(tx) {
+    return tx.ledger_index;
+  });
+
+  async.each(_.keys(groupedTx), function(ledger, next) {
```

Comment:
The violation is this one: https://github.com/ripple/javascript-style-guide/blob/master/eslintrc#L123